packages/roosterjs-editor-plugins/lib/plugins/TableCellSelection/TableCellSelection.ts

```diff
@@ -91,6 +92,9 @@ export default class TableCellSelection implements EditorPlugin {
     onPluginEvent(event: PluginEvent) {
         if (this.editor) {
             switch (event.eventType) {
+                case PluginEventType.EditorReady:
+                    this.editor.queryElements('table', table => table.removeAttribute('id'));
```

Comment:
Suggest we don't remove existing id. Only if we found duplicated id, we can change them.

Reply:
Duplicated id can be created any time.

Suggest we do the duplicate check when start to select. e.g.

```ts
function createUniqueId() {
  if (!node.id) { node.id = .... }
  else if (editor.queryElements('#' + node.id).length > 1) {
    // Change to another id since it is duplicated
  }
}
 ```

Reply:
Sounds good, fixed in the latest push, I think we can just remove the ID if duplicated in the TableCell Plugin.
If we need to assign a new Id we can do it in the selectTable Api.